app/controllers/v0/facilities/va_controller.rb

```diff
@@ -53,12 +128,32 @@ def validate_no_services_without_type
   end
 
   def validate_type_and_services_known
+    return if params[:type] == 'cc_provider'
     raise Common::Exceptions::InvalidFieldValue.new('type', params[:type]) unless
       BaseFacility::TYPES.include?(params[:type])
     unknown = params[:services].to_a - BaseFacility::SERVICE_WHITELIST[params[:type]]
     raise Common::Exceptions::InvalidFieldValue.new('services', unknown) unless unknown.empty?
   end
 
+  def format_provloc(provider, prov_info)
```

Comment:
This feels like something the provider object should own

Reply:
I was able to reconcile the slightly different structures from the locator and provider details endpoint and use the provider model/serializer for this